server/server.go

```diff
@@ -90,16 +83,15 @@ func NewRodeServer(
 }
 
 type rodeServer struct {
-	pb.UnimplementedRodeServer
 	logger              *zap.Logger
 	esClient            *elasticsearch.Client
 	filterer            filtering.Filterer
 	grafeasCommon       grafeas_proto.GrafeasV1Beta1Client
 	grafeasProjects     grafeas_project_proto.ProjectsClient
-	opa                 opa.Client
 	elasticsearchConfig *config.ElasticsearchConfig
 	resourceManager     resource.Manager
 	indexManager        indexmanager.IndexManager
+	policy.Manager
```

Comment:
This embeds the policy.Manager struct, which is implementing a portion of the larger `pb.RodeServer` interface. 
Is this a pattern we want to adopt? It requires that we don't embed `pb.UnimplementedRodeServer` -- IMHO this is fine, since the use case for that option is aimed more at servers that are shared across applications (for example, the gRPC health server)

Reply:
I think this makes sense, so we don't have a bunch of methods within `server` that act as proxies to the relevant package.  I did this a bit with the resource methods and wasn't happy with it, so this should help remove some unnecessary code